.pre-commit-config.yaml

```diff
@@ -42,17 +42,20 @@ repos:
     - id: flake8
       additional_dependencies: [flake8-bugbear==21.3.1]
 
-  - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.910-1
-    hooks:
-    - id: mypy
-      args: [--config-file=setup.cfg]
-      additional_dependencies:
-      - myst-parser~=0.14.0
-      files: >
-        (?x)^(
-            myst_nb/parser.py|
-        )$
+  # - repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
Can you add a comment explaining why this section is commented out, and when we should un-comment it? (or, can we just delete it?)

Reply:
fixed